Gems/PhysX/Code/Source/ForceRegion.cpp

```diff
@@ -236,16 +236,21 @@ namespace PhysX
     {
         EntityParams entityParams;
         entityParams.m_id = entityId;
-        AZ::TransformBus::EventResult(entityParams.m_position
-            , entityId
-            , &AZ::TransformBus::Events::GetWorldTranslation);
-        Physics::RigidBodyRequestBus::EventResultReverse(entityParams.m_velocity
-            , entityId
-            , &Physics::RigidBodyRequestBus::Events::GetLinearVelocity);
-        Physics::RigidBodyRequestBus::EventResultReverse(entityParams.m_mass
-            , entityId
-            , &Physics::RigidBodyRequestBus::Events::GetMass);
+
+        AzPhysics::RigidBody* rigidBody = nullptr;
+        Physics::RigidBodyRequestBus::EventResultReverse(rigidBody, entityId, &Physics::RigidBodyRequestBus::Events::GetRigidBody);
```

Comment:
question: Just curious why this is `EventResultReverse` and not `EventResult` 🤔 (I know it was like this before but it just seemed unusual to me and I wondered what the reason was).